manifests/frontend/install.pp

```diff
@@ -44,12 +45,25 @@
     }
   }
 
+  $_config_file = '/opt/apache-tomcat/bin/setenv.sh'
+  if ! defined(Concat[$_config_file]) {
+    concat { $_config_file:
+      owner          => $::tomcat::user,
```

Comment:
because you use $::tomcat::user, you want `require ::tomcat` instead of `include ::tomcat`